src/index.html

```diff
@@ -11,7 +11,159 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="http://localhost:3000/api"
+     method="post"
+     autocomplete="off"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">
+        Surname:
+          <input
+            name="feedback"
+            rows="1"
+            cols="20"
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+            name="feedback"
+            rows="1"
+            cols="20"
+          >
+        </label>
+        <label class="form-field">
+        How old are You?
+          <input
```

Comment:
```suggestion
        <label class="form-field">
          How old are You?
          <input
```
check and fix other places of wrong indentation

Reply:
still not fixed